var/spack/repos/builtin/packages/papi/package.py

```diff
@@ -72,6 +72,7 @@ class Papi(AutotoolsPackage, ROCmPackage):
     conflicts("%gcc@8:", when="@5.3.0", msg="Requires GCC version less than 8.0")
     conflicts("+sde", when="@:5", msg="Software defined events (SDE) added in 6.0.0")
     conflicts("^cuda", when="@:5", msg="CUDA support for versions < 6.0.0 not implemented")
+    conflicts("%cce", when="@7.1", msg="-ffree-form flag not recognized")
```

Comment:
```suggestion
    conflicts("%cce", when="@7.1:", msg="-ffree-form flag not recognized")
```
Unless we think this'll be removed/fixed the the next version, I might suggest conflicting with `%cce` until it is resolved (unlimited upper bound) so version bumps don't break CI.